cmd/launcher/options.go

```diff
@@ -22,6 +22,19 @@ const (
 	skipEnvParse         = runtime.GOOS == "windows" // skip environmental variable parsing on windows
 )
 
+// Adapted from
+// https://stackoverflow.com/questions/28322997/how-to-get-a-list-of-values-into-a-flag-in-golang/28323276#28323276
+type arrayFlags []string
+
+func (i *arrayFlags) String() string {
+	return strings.Join(*i, " ")
```

Comment:
(nit) Do we call this? And if we do, is `","` more appropriate? (I don't see it called)

Reply:
We don't call it, it's just required for the interface. I don't actually know when it would ever be renderd by this method.